grafana/overlays/nerc-ocp-obs/grafanas/grafana.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: integreatly.org/v1alpha1
+kind: Grafana
+metadata:
+  name: grafana
+  namespace: grafana
+spec:
+  deployment:
+    envFrom:
+      - secretRef:
+          name: oauth-client-secret
+      - secretRef:
+          name: grafana-serviceaccount-token
+      - configMapRef:
+          name: openshift-service-ca.crt
+  config:
+    server:
+      root_url: https://grafana.apps.nerc-ocp-infra.rc.fas.harvard.edu
```

Comment:
We'll want to change `infra` to `obs` here I think.

Reply:
```suggestion
      root_url: https://grafana.apps.nerc-ocp-obs.rc.fas.harvard.edu
```